torch/csrc/jit/export.cpp

```diff
@@ -234,21 +234,35 @@ void encodeModel(onnx::ModelProto* p_m, const std::shared_ptr<Graph>& g,
   encodeGraph(p_g, g, initializers);
 }
 
+namespace {
+std::string getNodeStackTraceString(Node* n) {
```

Comment:
Should this go somewhere more well-known to encourage more use of error reporting?